test/test-signatures.js

```diff
@@ -41,7 +41,7 @@ describe('Signatures', function() {
 
   describe('Signatures on saml:Response - Only 1 saml:Assertion', () => {
     //== VALID
-    it('R1A - both signed => valid', testOneResponse('/valid/response.root-signed.assertion-signed.xml', false, 1));
+    it('R1A - both signed => valid', testOneResponse('/valid/response.root-signed.assertion-signed.xml', false, 2));
     it('R1A - root signed => valid', testOneResponse('/valid/response.root-signed.assertion-unsigned.xml', false, 1));
```

Comment:
I'm not sure why adding newlines to the SAMLResponse causes this change to be needed, but it does

Reply:
Since this is changing a test, which is basically our contract to our consumers that we haven't broken anything or change any behavior unexpectedly, we really should know why this value much change, or we'll have to do a breaking change release. Even then, we should know _why_ the change is needed.

Reply:
Nevermind, I see why. I did not update the signature value in the fixture correctly.